### DISCUSSION TITLE
ブロック機能に「相手からの、ブロック以前に書き込まれたコメント削除」を追加してほしい。

### DISCUSSION THREAD

**Discussion by yuki-2000** [increments/qiita-discussions#159] [要望・アイデア・不具合]
普段Qiitaで記事を書いているのですが、スパムまではいかないのですが、明らかにプログラミングの知識が足りない人が永遠とコメント欄で質問してくることがあります。おそらく本人に悪気はないのですが、よくわからないことをコメントしてきます。
このままでは記事を見に来たほかの利用者がコメントしづらかったり、有益な情報を得られないなど記事の質の低下につながると思います。
とりあえずQiitaでブロックしたため今後コメントされる恐れはなくなりましたが、記事にあるブロック以前に書き込まれたコメントを削除したいです。
ブロックの機能に内蔵して一括で削除するか、記事の投稿者がコメントを選んで個別に削除できるようにしてほしいです。

また、問題のあるユーザーを無視していたのですが、ついにはGithubのpuclicの関係ないリポジトリのissueで質問するようになりました。ブロックしているユーザーからは、私のユーザページから連携しているtwitter、githubなどを見れないようにすることは可能でしょうか。

よろしくお願いします。

**Comment by kskwtnk** [comment]
@yuki-2000 
ご意見ありがとうございます！

申し訳ありませんが、以下の2つは現時点では検討しておりません 🙇 

- 記事投稿者によるコメントの削除
- ブロックしているユーザーから連携しているTwitter, GitHubを見れないようにする

というのも……

- 今回のように一律で表示して欲しくない場合もあれば、そうでない場合も有り得る
- ブロックした人から見えないだけでなく、コメントそのものを削除してしまうのは越権行為となりそう

感情としては共感できる内容なのですが、運営の提供する機能や振る舞いとしては既存のブロック機能の範疇でご対応いただきたいです
また、もしコメントがコミュニティガイドラインなどに違反してましたら、お手数ですが通報をしていただけると幸いです

やや煮え切らない回答になってしまい申し訳ありませんがご理解いただけますと幸いです 🙇

**Comment by yuki-2000** [comment]
返信ありがとうございます。

コメントそのものを削除するのは難しいのですね。
コメントがコミュニティガイドラインなどに違反していたら通報するようにします。

ありがとうございます。
